### ISSUE TITLE
Read collection name from @document value in addition to collection attribute. 

### ISSUE THREAD

**Issue by xetra11** [kaiso/relmongo#63]
While debugging #62 I found some issue when not using `@Document(collection="cars")`
In `PersistentPropertySavingCallback` Line 115
```java
String collection = ReflectionsUtil.getGenericType(field).getAnnotation(Document.class).collection();
```
Only a getter for `collection()` is used. However like in my example you can define the collection name also by omitting `document=` and just using the value attribute.

```kotlin
@Document("cars")
data class Car(
    @Id
    val id: String,
    val name: String,
)
```
The logic therefore ignores the collection name. Unfortunately it does not solve the problem I mentioned above

**Comment by kaiso** [comment]
Hello @xetra11 thank you for reporting the issue, it will be fixed in the coming release.

**Comment by xetra11** [comment]
@kaiso I already created a pull request

**Comment by kaiso** [comment]
@xetra11 Fixed in version [3.4.1](https://github.com/kaiso/relmongo/releases/tag/v3.4.1)
please feel free to report new issues if any and thank you for using **RelMongo**
